website/pages/home/index.jsx

```diff
@@ -1,350 +1,294 @@
-            decisions, and visualize failures before they happen.
-          </p>
-          <CodeSample>{`<span class='green'>admin@hashicorp.com:</span> nomad job plan example.nomad
-<span class='yellow'>+/-</span> <span class='bold'>Job: "example"</span>
-<span class='yellow'>+/-</span> <span class='bold'>Task Group: "cache"</span> (<span class='green'>2 create</span>, <span class='blue'>1 in-place update</span>)
-  <span class='yellow'>+/-</span> Count: "1" => "3" (<span class='green'>forces create</span>)
-      Task: "redis"
-
-<span class='bold'>Scheduler dry-run:</span>
-- <span class='green bold'>All tasks successfully allocated.</span>
-
-<span class='bold'>Job Modify Index: 7</span>
-
-<span class='green'>admin@hashicorp.com</span>: nomad job run -check-index 7 example.nomad
-==> Monitoring evaluation "e338a6ae"
-    Evaluation triggered by job "example"
-    Evaluation within deployment: "43310206"
-    Allocation "991a0d89" created: node "c5916a99", group "cache"
-    Allocation "daee8be1" created: node "c5916a99", group "cache"
-    Allocation "83f29312" modified: node "c5916a99", group "cache"
-    Evaluation status changed: "pending" -> "complete"
-==> Evaluation "e338a6ae" finished with status "complete"
-        `}</CodeSample>
+      </div>
 
-          <h3 className="g-type-display-4">View Application Logs</h3>
-          <p className="g-type-body">
-            Stream application logs directly in the terminal to help analyze and
-            debug applications.
-          </p>
-          <CodeSample>{`<span class='green'>admin@hashicorp.com:</span> nomad alloc logs -tail -job example
-20 Dec 16:29:47 # Server started, Redis version 3.2.9
-20 Dec 16:29:47* The server is now ready to accept connections on port 6379
-        `}</CodeSample>
+      <LearnNomad
+        items={[
+          {
+            title: 'Getting Started',
+            category: 'Step-by-Step Guides',
+            time: '24 mins',
+            link:
+              'https://learn.hashicorp.com/nomad?track=getting-started#getting-started',
+            image: require('./img/learn-nomad/cap.svg')
+          },
+          {
+            title: 'Deploy and Manage Nomad Jobs',
+            category: 'Step-by-Step Guides',
+            time: '36 mins',
+            link:
+              'https://learn.hashicorp.com/nomad?track=managing-jobs#getting-started',
+            image: require('./img/learn-nomad/cubes.svg')
+          }
+        ]}
+      />
 
-          <div className="centered">
-            <h5 className="g-type-body-large">
-              The introduction contains a walkthrough guide, glossary, and a
-              range of examples for learning and experimenting with Nomad.{' '}
-            </h5>
-            <Button
-              url="/intro"
-              title="Get Started"
-              theme={{ brand: 'nomad' }}
-            />
-          </div>
-        </div>
-      </section>
-    </div>
-  )
-}
+      <NomadEnterpriseInfo />
```

Comment:
Might be slightly of of scope for this PR, but since I'm in copy editing mode - this headline isn't actually a question. Should probably drop the `?` or rephrase ("When does Nomad Enterprise make sense?")

<img width="1532" alt="Screenshot 2020-02-23 10 34 08" src="https://user-images.githubusercontent.com/845983/75114893-15a46f80-5628-11ea-9b38-5439b2a7b5b3.png">

Reply:
I think out of scope for the initial launch, but I'll have another conversation with the team here.

I actually think there was some decent thought put into the wording here, so for now I will assume this is OK, but will double back to make sure.